app/src/main/java/com/example/imageEditor2/module/NetworkModule.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.imageEditor2.module
+
+import android.content.Context
+import android.preference.PreferenceManager
+import com.example.imageEditor2.Api
+import com.example.imageEditor2.ApiAuthorize
+import com.example.imageEditor2.utils.BASE_URL
+import com.example.imageEditor2.utils.BASE_URL_API
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+object NetworkModule {
+    var mToken = ""
+    private const val TIME_OUT: Long = 10
+    private var mLanguage = "vi"
+
+    internal fun providePostApi(retrofit: Retrofit): Api {
+        return retrofit.create(Api::class.java)
+    }
+
+    internal fun provideRetrofitInterface(context: Context): Retrofit {
+        val mPrefs = PreferenceManager.getDefaultSharedPreferences(context)
+
+        if (mToken == "") {
+            mToken = mPrefs.getString("PREF_ACCESS_TOKEN", "") ?: ""
+        }
+
+        val httpClient = OkHttpClient.Builder()
+        httpClient.connectTimeout(TIME_OUT, TimeUnit.SECONDS)
+        httpClient.readTimeout(TIME_OUT, TimeUnit.SECONDS)
+
+        httpClient.addInterceptor { chain ->
+            val request =
+                chain.request().newBuilder()
+                    .addHeader("Content-Type", "application/json")
```

Comment:
```suggestion
                    .addHeader(CONTENT_TYPE, "application/json")
```
Hard code & đang duplicate vs bên dưới ref: [link](https://github.com/awesome-academy/HN_Android_2024_Image_Editor_2/pull/2/files#diff-76e58641e721fa3662a0825135ab04ca38878c9cee95687629405be7b933df3dR69)

sửa tương tự vs các giá trị khác, hạn chế hard code